Login and signup enhancement

elezersolomon opened the pull request.

elezersolomon committed:
feat(login and signup): login page enhacement and full sign up page

validation and ui improvement for login page, funn sign up page with code entry popup and redirect
to login page

elezersolomon committed:
added .git ignore to .gitignore

elezersolomon committed:
feat(adds tab template and enhances login and signup functionalities): ui enhancement & feature

removed headers for signup and login pages as well as new ones and added pages with tabs after login

tutushtutush commented on .gitignore:
why was this file deleted?

elezersolomon replied:
git ignore was included on git ignore it-self. i have removed it.

tutushtutush commented on app/(screens)/signUp.tsx:
No need for a sign up page for now.

elezersolomon replied:
sign up page has been removed.

tutushtutush reviewed (changes requested):
please check the comments

elezersolomon committed:
docs(file inclusion): included git ignore

git ignore was included in git ignore it self and was untracked so i removed git ignore from git
ignore it self.

elezersolomon committed:
Update .gitignore

elezersolomon committed:
refactor(remove feature): removed signup feature

removed sign up feature as per comment on the current pull request.

#5, #6, #7
